ibm_catalog.json

```diff
@@ -46,8 +46,8 @@
       "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [https://github.com/terraform-ibm-modules/terraform-ibm-account-infrastructure-base/issues](https://github.com/terraform-ibm-modules/terraform-ibm-account-infrastructure-base/issues). Please note this product is not supported via the IBM Cloud Support Center.",
       "flavors": [
         {
-          "label": "Fully configurable",
-          "name": "fully-configurable",
+          "label": "Resource groups with account settings",
```

Comment:
@MatthewLemmond FYI i remove the caps on "account settings" - there no reason for them to be capitals, its not a service name